MonoGame.Framework/CurveKeyCollection.cs

```diff
@@ -20,6 +20,7 @@ public class CurveKeyCollection : ICollection<CurveKey>
         #region Private Fields
 
         private readonly List<CurveKey> _keys;
+        private readonly IComparer<CurveKey> _keyPositionComparer = Comparer<CurveKey>.Create((x, y) => x.Position.CompareTo(y.Position));
```

Comment:
CurveKey already implements IComparable<CurveKey> which is faster. There's no need to use a IComparer delegate.
https://github.com/MonoGame/MonoGame/pull/4172#issuecomment-147288961

Reply:
Yeah I see now that the position comparison is already implemented. :)